README.md

```diff
@@ -2,28 +2,75 @@
 
-Merge Gatekeeper was created to provide more control over merges.
+**Merge Gatekeeper** was created to provide more control for merges. By placing Merge Gatekeeper to run for all PRs, it can check all other CI jobs that get kicked off, and ensure all the jobs are completed successfully. If there is any job that has failed, Merge Gatekeeper will fail as well. This allows merge protection based on Merge Gatekeeper, which can effectively ensure any CI failure will block merge. All you need is the Merge Gatekeeper as one of the PR based GitHub Action, and set the branch protection rule as shown below.
 
----
+![Branch protection example](/assets/images/branch-protection-example.png)
+
+We are looking to add a few more features, such as extra signoff from non-coder, label based check, etc.
 
-NOTE <sup>(\*1)</sup>: There are some other hacks, such as using an empty job with the same name to override the status, but those solutions do not provide the flexible control we are after.
+<sup><sub>NOTE:  
+<sup>(\*1)</sup> There are some other hacks, such as using an empty job with the same name to override the status, but those solutions do not provide the flexible control we are after.</sub></sup>
 
 <!-- == imptr: background / end == -->
 
+You can find [more details here](/docs/details.md).
+
+## 🚀 How Can I Use Merge Gatekeeper?
+
+<!-- == imptr: example-usage / begin from: ./docs/action-usage.md#[simple-usage] == -->
+
+The easiest approach is to copy the standard definition, and save it under `.github/workspaces` directory. There is no further modification required unless you have some specific requirements.
+
+#### With `curl`
+
+```bash
+curl -sSL https://raw.githubusercontent.com/upsidr/merge-gatekeeper/main/example/merge-gatekeeper.yml \
+  > .github/workflows/merge-gatekeeper.yml
+```
+
+#### Directly copy YAML
+
+The below is the copy of [`/example/merge-gatekeeper.yml`](/example/merge-gatekeeper.yml).
+
+<!-- TODO: replace below using Importer once Importer supports code block wrapping
+     == imptr: basic-yaml / begin from: ./example/definitions.yaml#[standard-setup] == -->
+
+```yaml
 ---
+name: Merge Gatekeeper
 
-You can find [more details here](/docs/details.md).
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  merge-gatekeeper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Merge Gatekeeper
+        uses: upsidr/merge-gatekeeper@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+<!-- == imptr: example-usage / end == -->
+
+You can find [more details here](/docs/action-usage.md).
 
 ## 🧪 Action Inputs
 
+There are some customisation available for Merge Gatekeeper.
```

Comment:
customization?

Reply:
This is a British spelling, so it is both correct :P